src/neo/SmartContract/StorageContext.cs

```diff
@@ -4,5 +4,6 @@ internal class StorageContext
     {
         public UInt160 ScriptHash;
         public bool IsReadOnly;
+        public UInt160 PreDataKey;
```

Comment:
Any other idea for the name of this variable?

Reply:
Is `preFix`  well? @vncoelho

Reply:
How about changing the `ScriptHash` to `ContractHash`, `PreDataKey` to `NameSpace`, `Preifx`, 
 or `StorageKeyPrefix` ?